### DISCUSSION TITLE
Multiple dashboards

### DISCUSSION THREAD

**Discussion by theusf** [Kong/insomnia#3159] [Ideas]
Currently i'm using insmonia on my personal studies and my work and all my collections are mixed on the dashboard, i was thinking that is a good ideia to have the possibility to have more dashboards so i can make an dash for my work and another for my personal collections.
Thanks!

**Comment by develohpanda** [comment]
Hey @theusf! We have some work coming up called Spaces which is related to sync and teamwork, but I think that can be extended in the future to allow for offline spaces, which are basically what you're describing - a way to group multiple collections into individual spaces.
